### ISSUE TITLE
Thaelis the Hungerer

### ISSUE THREAD

**Issue by xnemeyethx** [dalaranwow/dalaran-wow#5650]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Thaelis is outside the building and do not have any adds to suport him in fight. He was put there cuz he just dissapeared in textures when pulled by a player but maybe it is working right now?
**Expected behaviour**: Tell us what should happen instead
Thaelis should be in the building in front of the bonfire. 
**Steps to reproduce the problem**:

1. Go to Eversong Woods, Farstrider Square.
2. Search for Thaelis in the ruins of Silvermoon City.
3. 

**Include proofs for this behaviour**
https://www.youtube.com/watch?v=6KZzv4EVcO8

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
https://wotlk.evowow.com/?npc=15949
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
